elephant/spectral.py

```diff
@@ -523,6 +529,352 @@ def multitaper_psd(signal, n_segments=1, len_segment=None,
     return freqs, psd
 
 
+def multitaper_cross_spectrum(signals, fs=1., n_segments=1, len_segment=None,
+                              frequency_resolution=None, overlap=0.5, nw=4.,
+                              num_tapers=None, peak_resolution=None,
+                              return_onesided=True, axis=-1):
+    """
+    Estimates the cross spectrum of a given 'neo.AnalogSignal' using the
+    Multitaper method.
+
+    The cross spectrum is obtained through the following steps
+
+    1. Cut the given data into several overlapping segments. The degree of
+       overlap can be specified by parameter `overlap` (default is 0.5,
+       i.e. segments are overlapped by the half of their length).
+       The number and the length of the segments are determined according
+       to the parameters `n_segments`, `len_segment` or `frequency_resolution`.
+       By default, the data is not cut into segments;
+
+    2. Calculate 'num_tapers' approximately independent estimates of the
+       spectrum by multiplying the signals with the discrete prolate spheroidal
+       functions (also known as Slepian function) and calculate the cross
+       spectrum for the tapered segment.
```

Comment:
I would revise this as:
Obtain approximately independent estimates of the spectrum for each signal segment by multiplying the segment with tapering functions (obtained as the discrete prolate spheroidal functions, also known as Slepian functions) and calculating the cross spectrum for the tapered segment. The number of tapering functions (and hence the number of the estimates) is specified by the parameter `num_tapers`.

Reply:
Addressed in [1cd45ee](https://github.com/INM-6/elephant/commit/1cd45ee7ed2e452c71b9d6049d802fcd1a30e77e)